tests/regression/roles/sonic_l2_interfaces/defaults/main.yml

```diff
@@ -34,31 +34,31 @@ tests:
         trunk:
           allowed_vlans:
             - vlan: 501
-            - vlan: 502
```

Comment:
Please revert the changes in this file. With the changes to the included changes to the l2 interfaces "facts" file, the consecutive vlan test cases should pass and they should be included in the regression.